exporters/trace/zipkin/zipkin.go

```diff
@@ -166,19 +166,12 @@ func (e *Exporter) ExportSpans(ctx context.Context, batch []*export.SpanData) er
 	if err != nil {
 		return e.errf("request to %s failed: %v", e.url, err)
 	}
-	e.logf("zipkin responded with status %d", resp.StatusCode)
+	defer resp.Body.Close()
```

Comment:
The original change to read the whole response body was included so the default HTTP client's Transport will
reuse HTTP/1.x "keep-alive" TCP connections. It seems like a good idea to keep this behavior.

https://github.com/open-telemetry/opentelemetry-go/issues/786#issuecomment-637633843

Reply:
Even if that is the case, the main feedback should come from http status code. If you want to, we can read the body (and discard it) in a defer. Does that make sense?

Reply:
Verification of the status code is a good addition :+1:. I would prefer to preserve the existing body handling and error logging.